deepchem/feat/base_classes.py

```diff
@@ -202,7 +212,7 @@ def featurize(self,
 
     return np.asarray(features)
 
-  def _featurize(self, complex: Tuple[str, str]):
+  def _featurize(self, datapoint: Optional[Tuple[str, str]] = None, **kwargs):
```

Comment:
Perhaps we should just add a return type here since logically `_featurize` out to return something. We might have to modify the parent `_featurize` as well in that case

Reply:
I tried to add a return type to `_featurize() `but it doesn't seme to resolve mypy conflicts in `grid_featurizer`